src/sonic-yang-models/tests/yang_model_tests/tests_config/portchannel.json

```diff
@@ -125,5 +125,63 @@
                 ]
             }
         }
+    },
+    "PORTCHANNEL_INTERFACE_LIST_VRF_TEST": {
+        "sonic-vrf:sonic-vrf": {
+            "sonic-vrf:VRF": {
+                "VRF_LIST": [{
+                    "vrf_name": "Vrf_blue"
+                }]
+            }
+        },
+        "sonic-portchannel:sonic-portchannel": {
+            "sonic-portchannel:PORTCHANNEL": {
+                "PORTCHANNEL_LIST": [{
+                    "name": "PortChannel0001",
+                    "admin_status": "up"
+                }]
+            },
+            "sonic-portchannel:PORTCHANNEL_INTERFACE": {
+                "PORTCHANNEL_INTERFACE_LIST": [{
+                    "name": "PortChannel0001",
+                    "vrf_name": "Vrf_blue"
```

Comment:
I think, We need one more positive test where we VRF name is not specified

Reply:
Addressed